src/java/com/wolfssl/provider/jsse/WolfSSLTrustX509.java

```diff
@@ -460,6 +460,12 @@ public X509Certificate[] getAcceptedIssuers() {
         WolfSSLDebug.log(getClass(), WolfSSLDebug.INFO,
             "entered getAcceptedIssuers()");
 
+        if (store == null) {
+            WolfSSLDebug.log(getClass(), WolfSSLDebug.ERROR,
+                    "Trust Manager was not initialized");
+            return null;
```

Comment:
I think this needs to return an empty X509Certificate array, based on my reading of the X509TrustManager Javadocs:

https://docs.oracle.com/javase/8/docs/api/javax/net/ssl/X509TrustManager.html#getAcceptedIssuers--

States the return value as:

```
Returns:
a non-null (possibly empty) array of acceptable CA issuer certificates.
```

Reply:
Thanks Chris! Updated it.